packages/hash/frontend/src/blocks/page/Comments/CommentThread.tsx

```diff
@@ -113,14 +114,18 @@ export const CommentThread: FunctionComponent<CommentThreadProps> = ({
 
           <Collapse in={expanded}>
             {collapsedReplies.map((reply) => (
```

Comment:
Change requested: the replies appear in the wrong order – if I create a reply, it appears above older replies

Reply:
I am aware of this, waiting for the `createdAt` field to be implemented in graph API's entities. I've talked to @thehabbos007 and he told me it would be a part of this [task](https://app.asana.com/0/1201095311341924/1202918403128224/f) _(internal)_.

We could implement a custom createdAt property field for comment entities in the meanwhile but I think that would be wasted effort to add it just to simply remove it later.

Reply:
I created thsi [task](https://app.asana.com/0/1201095311341924/1203230918183050/f) _(internal)_ to track this.